workers/repomirrorworker/test/test_repomirrorworker.py

```diff
@@ -311,8 +311,8 @@ def skopeo_test(args, proxy):
 @pytest.mark.parametrize(
     "rollback_enabled, expected_delete_calls, expected_retarget_tag_calls",
     [
-        (True, ["deleted", "updated", "created"], ["updated"]),
-        (False, ["deleted"], []),
+        (True, ["deleted", "zzerror", "updated", "created"], ["updated"]),
+        (False, ["deleted","zzerror"], []),
```

Comment:
What's zzerror?

Reply:
Handles the case where the tag was still created for the errored image. In this case tag `zzerror` is a `skopeo copy` that fails.